app/api/resources/user.py

```diff
@@ -40,9 +40,9 @@ class UserList(Resource):
     @users_ns.doc("list_users", params={"search": "Search query"})
     @users_ns.doc(
         responses={
-            HTTPStatus.UNAUTHORIZED: f"{messages.TOKEN_HAS_EXPIRED['message']}<br>"
-            f"{messages.TOKEN_IS_INVALID['message']}<br>"
-            f"{messages.AUTHORISATION_TOKEN_IS_MISSING['message']}"
+            HTTPStatus.UNAUTHORIZED: f"{messages.TOKEN_HAS_EXPIRED}<br>"
```

Comment:
Also add this for OK status `@users_ns.response(HTTPStatus.OK, f"{messages.GENERAL_SUCCESS_MESSAGE}", public_user_api_model)`.
And similary do for others like Users: GET user/ , Users: GET /users/verified by adding appropriate user api model.